source/modules/MEI.xml

```diff
@@ -364,37 +364,49 @@
     <content>
       <valList type="closed">
         <valItem ident="dashed">
-          <desc xml:lang="en">Dashed line (Unicode 1D104).</desc>
+          <desc xml:lang="en">Dashed line (SMuFL E036 and Unicode 1D104).</desc>
         </valItem>
         <valItem ident="dotted">
-          <desc xml:lang="en">Dotted line.</desc>
+          <desc xml:lang="en">Dotted line (SMuFL E037).</desc>
         </valItem>
         <valItem ident="dbl">
-          <desc xml:lang="en">(Unicode 1D101).</desc>
+          <desc xml:lang="en">Double barline (SMuFL E031 and Unicode 1D101).</desc>
         </valItem>
         <valItem ident="dbldashed">
           <desc xml:lang="en">Double dashed line.</desc>
         </valItem>
         <valItem ident="dbldotted">
           <desc xml:lang="en">Double dotted line.</desc>
         </valItem>
+        <valItem ident="dblheavy">
+          <desc xml:lang="en">Heavy double barline (SMuFL E035).</desc>
+        </valItem> 
+        <valItem ident="dblsegno">
+          <desc xml:lang="en">Segno serpent with vertical lines (SMuFL E04B).</desc>
+        </valItem> 
         <valItem ident="end">
-          <desc xml:lang="en">(Unicode 1D102).</desc>
+          <desc xml:lang="en">End barline (SMuFL E032 and Unicode 1D102).</desc>
         </valItem>
+        <valItem ident="heavy">
+          <desc xml:lang="en">Heavy barline (SMuFL E034).</desc>
+        </valItem> 
         <valItem ident="invis">
           <desc xml:lang="en">Bar line not rendered.</desc>
         </valItem>
         <valItem ident="rptstart">
-          <desc xml:lang="en">Repeat start (Unicode 1D106).</desc>
+          <desc xml:lang="en">Repeat start (SMuFL E040 and Unicode 1D106).</desc>
         </valItem>
         <valItem ident="rptboth">
-          <desc xml:lang="en">Repeat start and end.</desc>
+          <desc xml:lang="en">Repeat start and end (SMuFL E042).</desc>
         </valItem>
         <valItem ident="rptend">
-          <desc xml:lang="en">Repeat end (Unicode 1D107).</desc>
+          <desc xml:lang="en">Repeat end (SMuFL E041 and Unicode 1D107).</desc>
+        </valItem>
+        <valItem ident="segno">
+          <desc xml:lang="en">Segno serpent (SMuFL E04A).</desc>
```

Comment:
I think we want U+E04B as double barline with segno glyph.

Reply:
When discussing it it seemed appropriate also to include the segno serpent that @musicEnfanthen pointed us to.

Reply:
but the E04B is not included now, just E04A (see https://www.smufl.org/version/latest/range/repeats/)? The last comment ("_also_") seems to contradict the code…

Reply:
E04A is just the serpent symbol without a barline, so why should we encode this as barline? 
I think the approach of SMuFL for this is to have it available for combination with barlines drawn with primitives, so e.g., in Verovio we would use E04A to put it on top of the drawn barlines. But for referencing the look in the schema we should only use E04B.

Reply:
@kepper and E04B _is_ in there with "dblsegno".

Reply:
From what I can tell, the serpent segno with and without double lines is used as an alternative to the barline. So it would make sense to have both. (This doesn't actually preclude *also* having serpent segnos as symbols on their own if they *don't* appear as a barline, too.)

Reply:
No, the one without lines is not an alternative to the barline. Though it can appear between notes on a staff you would not encode this as a barline. This should be encoded as a <segno> (see #520), which may appear on or above a staff.

Reply:
I think it is good to have `segno` just in case someone  encounters a single barline with a segno. I removed the references to SMuFL because that can be misleading.